test/karma.conf.js

```diff
@@ -13,9 +13,11 @@ module.exports = function(config) {
     }
   };
 
-  // On Travis CI, we can only run in Firefox.
+  // TODO: This should include firefox. It is currently turned off because
+  //       of https://github.com/travis-ci/travis-ci/issues/8242 When this issue is
+  //       resolved, this should be updated back to latest
```

Comment:
minor: but in this scenario it should just say `this should be updated to include firefox` instead of the mention of latest

Reply:
Or just remove everything after the issue link